docs/installing-solidity.rst

```diff
@@ -28,7 +28,7 @@ download the .ZIP file as explained on that page.
 npm / Node.js
 =============
 
-This is probably the most portable and most convenient way to install Solidity locally.
+We recommend installing with npm, it is the most portable and convenient way to install Solidity.
```

Comment:
I think we have to be more careful as it is not the best tool right now.

Simplest one is to use Remix in a browser. Next, in terms of installation complexity, npm or any other distribution specific package is the same (brew, apt-get, snap, pacman).

Reply:
Here's an alternate proposal: #3381